public/static-checkout/generate_static_pages.py

```diff
@@ -1,17 +1,17 @@
 import csv
 # Combine minified html/css/js into one string
 with open('dist/minified.html', 'r') as file:
-    html = file.read().replace('\n', '')
```

Comment:
why was this removed?

Reply:
They appear to be unnecessary in the minified case (based on my testing) and they interfere with generating non-minified pages for debugging.